### DISCUSSION TITLE
login problem

### DISCUSSION THREAD

**Discussion by asifrana1972** [horilla-opensource/horilla#233] [Q&A]
If I create a blank database, how login in horilla system. The system is blank.

**Comment by horilla-opensource** [comment]
Hi @asifrana1972 ,
If you are having the latest code base, and you are starting with a blank database without creating a super user, you'll get an option in the login page to _initialize database_ through which you can create a super user and add the necessary details for initial setting up.
For that first you'll have to setup a _db_init_password_ in the _horilla/horilla_settings.py_ file and use that password while setting up the database.

With Regards,
**Team Horilla**
